docs/io/fits/index.rst

```diff
@@ -530,6 +530,26 @@ encapsulate it in an :class:`HDUList` object first.
 Creating a New Table File
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 
+.. note::
+
+    If you want to create a simple binary FITS table with no other HDUs,
```

Comment:
Does anyone use ASCII table in FITS? :stuck_out_tongue_closed_eyes:  (Sure, I can highlight it.)